ServerCore/Pages/Teams/Hints.cshtml.cs

```diff
@@ -10,6 +11,8 @@
 
 namespace ServerCore.Pages.Teams
 {
+    [Authorize(Policy = "PlayerCanSeePuzzle")]
```

Comment:
Were you able to verify that having these two attributes is an AND relationship? I thought it was an OR (meaning that now either being able to see the puzzle or being on the team would allow access)

Reply:
Yes I was :)